ksqldb-functional-tests/src/test/resources/query-validation-tests/source-stream.json

```diff
@@ -0,0 +1,37 @@
+{
+  "tests": [
+    {
+      "name": "create source stream",
+      "statements": [
+        "CREATE SOURCE STREAM INPUT (K STRING KEY, text STRING) WITH (kafka_topic='test_topic', value_format='DELIMITED');",
+        "CREATE STREAM OUTPUT AS select * from INPUT;"
```

Comment:
Could we add tests that attempt to insert data into a source stream and fails?

Reply:
Added